massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -1080,6 +1100,11 @@ impl ProtocolWorker {
 
         Ok(())
     }
+
+    /// New produced operations.
+    pub(crate) fn subscribe_new_operations(&self, sink: SubscriptionSink) {
+        broadcast_via_ws(self.ws_config.operation_sender.clone(), sink);
```

Comment:
I don't think making a function for a line of code it's worth it.

Reply:
this call will be replaced by: 

```rust
/// Brodcast the stream(sender) content via a WebSocket
pub fn broadcast_via_ws<T: Serialize + Send + Clone + 'static>(
    sender: Sender<T>,
    mut sink: SubscriptionSink,
) {
    let rx = BroadcastStream::new(sender.subscribe());
    tokio::spawn(async move {
        match sink.pipe_from_try_stream(rx).await {
            SubscriptionClosed::Success => {
                sink.close(SubscriptionClosed::Success);
            }
            SubscriptionClosed::RemotePeerAborted => (),
            SubscriptionClosed::Failed(err) => {
                sink.close(err);
            }
        };
    });
}
```

Reply:
If we do this, the `massa-ws` module will be removed.

Reply:
I'm saying that `broadcast_via_ws` is already a function and making the function `subscribe_new_operations` that only have one line of code seems to be an overhead.

Reply:
done